### PR TITLE
feat: enhanced appinsights

### PR DESCRIPTION
**Pull request by naren-bellala**
#### 📲 What

A description of the change.

#User Story 788: Application Insights enhancements

#### 🤔 Why
		
Why it's needed, background context.
		
#### 🛠 How
		
More in-depth discussion of the change or implementation.

#### 👀 Evidence
		
Screenshots / external resources / links / etc.
Link to documentation updated with changes impacted in the PR
		 
#### 🕵️ How to test

Notes for QA

#### ✅ Acceptance criteria Checklist

- [ ] Code peer reviewed?
- [ ] Documentation has been updated to reflect the changes?
- [ ] Passing all automated tests, including a successful deployment?
- [ ] Passing any exploratory testing?
- [ ] Rebased/merged with latest changes from development and re-tested?
- [ ] Meeting the Coding Standards?

### CODE REVIEW

File: packages/template-cli/templates/src/ssr/package.json [hunk 1]
```diff
@@ -1,5 +1,6 @@
 {
-  "name": "webapp",
+  "name": "PROJECT_NAME",
+  "version": "0.1.0",
```
Comment (estherthetesteramido) [inline packages/template-cli/templates/src/ssr/package.json:3]: This shouldn't have a version.

File: packages/template-cli/templates/src/ssr/package.json [hunk 2]
```diff
@@ -1,5 +1,6 @@
 {
-  "name": "webapp",
+  "name": "PROJECT_NAME",
```
Comment (estherthetesteramido) [inline packages/template-cli/templates/src/ssr/package.json:2]: @dnitsch are these going to be replaced from the CLI?

File: packages/template-cli/templates/src/ssr/package.json (hunk 1 above)
Comment (estherthetesteramido) [reply packages/template-cli/templates/src/ssr/package.json:3]: Ah I see, we log in app insights with the version. Do we need to? Else, default to 0.0.0?

Comment (naren-bellala) [reply packages/template-cli/templates/src/ssr/package.json:3]: Yes. This is for app insights annotation.  What do we need to ? 

CSR is also versioned same way. So we are making it consistent

File: packages/template-cli/templates/src/ssr/package.json (hunk 2 above)
Comment (naren-bellala) [reply packages/template-cli/templates/src/ssr/package.json:2]: Yes. Just as the previous comment

**Pull request merged**